app/models/pokemon.js

```diff
@@ -1,28 +1,39 @@
+"use strict";
+
 var mongoose = require('mongoose');
-var Schema   = mongoose.Schema;
+var Schema = mongoose.Schema;
 
+var pokemonGoBasicSchema = new Schema({
+    Number: {type: String},
+    Name: {type: String},
+    Classification: {type: String},
+    Types: [String],
+    Resistant: [String],
+    Weaknesses: [String],
+    Fast_Attacks: [{Name: {type: String}, Type: {type: String}, Damage: {type: Number}}],
```

Comment:
For schema you can use camel-casing as we have followed the same in pokemonSighting. For e.g., Fast_Attacks to fastAttacks